docs/.gitignore

```diff
@@ -11,5 +11,7 @@ themes/thxvscode/assets/js/node_modules/
 
 static/playground/
 
-# Images in these folder are autogenerated at docs build time
-content/docs/*/images/**/*.png
+# Images at these paths are autogenerated at docs build time
+content/community/doc-system/*.png
+static/images/architecture.png
```

Comment:
seems like generated images here should go in their own folder so they don't need to all be explicitly added here (also having generated images under the static folder seems kind of off?)

Reply:
`static` is [what Hugo calls the folder](https://gohugo.io/content-management/static-files/) of stuff that gets copied as-is to the output directory. I could move the source ditaa files to a `generated` folder, but then you have to refer to the output images with that in the URL.